src/components/Editor/commands.tsx

```diff
@@ -97,3 +101,26 @@ const toggleCheckbox = (text: string): string => {
 export const toggleCheckList = (text: string): string => {
   return text.split('\n').map(toggleCheckbox).join('\n')
 }
+
+export const toggleCheckListAll = (text: string): string => {
+  return toggleCheckbox(text)
+}
+
+export const toggleTagBlocks = (text: string): ReactElement | string => {
+  if (text.startsWith('`') && text.endsWith('`')) {
+    console.log('in condition')
+    return (
+      <Code
```

Comment:
I see what you are trying to do here. But seems like this only modifies the text. It cannot change the underlying dom. Seems like we might have to look into creating a custom extension - https://discuss.codemirror.net/t/how-to-define-highlighting-styles-for-blocks/4029/2
or maybe through some custom syntax highlighting? src/components/Editor/customSyntaxHighlighting.ts